routes/base-db-methods.js

```diff
@@ -88,7 +88,7 @@ function list(Model) {
         queryFilter = processDatesRangeParameters(ctx);
       }
       try {
-        const docs = await Model.find(queryFilter, null, options);
+        const docs = await Model.find(queryFilter, ctx.query.fields, options);
```

Comment:
Are you sure it's not breaking anything?
@bsuro10 might know for sure.

Reply:
I'm pretty sure